Resources/Prototypes/Recipes/Lathes/Packs/medical.yml

```diff
@@ -62,6 +62,7 @@
   - ClothingHandsGlovesLatex
   - ClothingHandsGlovesNitrile
   - ClothingMaskSterile
+  - ClothingMaskBreathMedical # DeltaV
```

Comment:
make a MedicalClothingStaticDeltaV recipe pack and parent MedicalClothingStatic to it instead